docs/backend/README.md

```diff
@@ -81,18 +81,22 @@ You will also need to rebuild local running docker containers by running `make l
 If you add a third-party library (directly or indirectly) that does not support [python typing](https://docs.python.org/3/library/typing.html), then you may need to add an entry to [`mypy.ini`](../../mypy.ini) to let mypy know [not to expect type hints for that library](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-type-hints-for-third-party-library).
 
 
-## Adding new users to the staging db:
-* get username / password credentials from aws secrets manager (:
-  * `aws secretsmanager get-secret-value --secret-id aspen-config`
+## Adding new users to the staging/prod db:
+Create a new user to the auth0 covidtracker tenet, take note of auth0 user id
```

Comment:
Is there a wiki or something elsewhere on how to go through this process?

Reply:
```suggestion
Create a new user to the auth0 covidtracker tenant, take note of auth0 user id
```

Reply:
not really.. it's more of a like show someone new how to do it situation. I can try to add some docs for that tomorrow

Reply:
in our aspen wiki